resources/scripts/components/server/ServerConsole.tsx

```diff
@@ -21,50 +21,67 @@ const ServerConsole = () => {
     const isTransferring = ServerContext.useStoreState(state => state.server.data!.isTransferring);
     const eggFeatures = ServerContext.useStoreState(state => state.server.data!.eggFeatures, isEqual);
 
+    const fullscreen = window.location.href.includes('/console');
+
     return (
-        <ServerContentBlock title={'Console'} css={tw`flex flex-wrap`}>
-            <div css={tw`w-full lg:w-1/4`}>
-                <ServerDetailsBlock/>
-                {isInstalling ?
-                    <div css={tw`mt-4 rounded bg-yellow-500 p-3`}>
-                        <ContentContainer>
-                            <p css={tw`text-sm text-yellow-900`}>
-                                This server is currently running its installation process and most actions are
-                                unavailable.
-                            </p>
-                        </ContentContainer>
+        <ServerContentBlock title={'Console'} css={fullscreen ? tw`absolute h-screen w-screen left-0 top-0 px-0 mt-0` : tw`flex flex-wrap`}>
+            {fullscreen ?
```

Comment:
This code is incorrect, you've disabled all of the features such that a popped out console won't correctly alert the user to issues.

It might make sense to create a new component here that has the console and console feature stuff, and just render it twice on this screen, alone if fullscreen, or inside the correct block location otherwise.

Reply:
Or just use some CSS to make the console "full-screen" without hiding other elements, something like an absolutely position element using a portal so that it is overlaying everything, and just make the portal and styling deterministic based on full screen.

Basically you have a lot of very specific console logic spread out over files that really shouldn't contain that.